### DISCUSSION TITLE
Add create/init repository to main

### DISCUSSION THREAD

**Discussion by Vgentman** [sourcegit-scm/sourcegit#806] [Ideas]
I know it's a small thing, but it would be great if you could add a new repository initialization function to the main window. I apologize in advance if such functionality exists, but I couldn't find it.

**Comment by love-linger** [comment]
You can open a folder from `Repositories` page. If it is not a valid `git` repository, the `Initialize Repository` dialog will popups.

![image](https://github.com/user-attachments/assets/9954ba51-3061-4056-9c44-9ad4ca251784)

**Reply by Vgentman** [reply]
I didn't check the most obvious thing, sorry for wasting your time(
